### ISSUE TITLE
rasa calculation failed

### ISSUE THREAD

**Issue by vecos1990** [google-deepmind/alphafold3#55]
I just wish to inquire if anyone is also experiencing this "rasa calculation failed" and if there is a wrap around

**Comment by Augustin-Zidek** [comment]
Does your input have more than 26 chains or did you use more than one letter chain IDs for any of the entities?

**Comment by dailypartita** [comment]
i have same error when my input like this:
```
>HA
QIQLVQSGPELKKPGETVKISCKASGYNFTNYGMNWVKQAPGKGLKWMGWINTYTGEPTYADDFKGRFAFSLETSANSASLQINNLKTDDTATYFCARKRGDWFAYWGQGTLVTVSA
```
the sequence id only support one uppercase like "A"? When I use "A1" "A2" or "AB", "HA"  its turns error.

**Comment by Augustin-Zidek** [comment]
> i have same error when my input like this:
> 
> ```
> >HA
> QIQLVQSGPELKKPGETVKISCKASGYNFTNYGMNWVKQAPGKGLKWMGWINTYTGEPTYADDFKGRFAFSLETSANSASLQINNLKTDDTATYFCARKRGDWFAYWGQGTLVTVSA
> ```
> 
> the sequence id only support one uppercase like "A"? When I use "A1" "A2" or "AB", "HA" its turns error.

Correct, due to the MKDSSP format, only single letter chain names are currently supported in the RASA calculation pipeline. I will try to address this in January 2025.

**Comment by Augustin-Zidek** [comment]
Hello, I have fixed this in https://github.com/google-deepmind/alphafold3/commit/3d06f4bfad412be77dfa4c6c5f746c92ce8e0c41. Please update AlphaFold 3 past this commit and rebuild your Docker container to get the fix.
